source/RocketPlugin.cpp

```diff
@@ -1769,6 +1770,7 @@ void RocketPlugin::onLoad()
     gamemodes.push_back(std::shared_ptr<RocketGameMode>(new KeepAway(this)));
     gamemodes.push_back(std::shared_ptr<RocketGameMode>(new Tag(this)));
     gamemodes.push_back(std::shared_ptr<RocketGameMode>(new Juggernaut(this)));
+    gamemodes.push_back(std::shared_ptr<RocketGameMode>(new BoostMod(this)));
```

Comment:
Would change this to:
```suggestion
    gamemodes.push_back(std::make_shared<BoostMod>(this));
```
the other game modes will also be changed to this in the next update.